### DISCUSSION TITLE
It is a bug or incorrect usage? Unable to draw lines in 8.1.0 but ok in 7.4.2

### DISCUSSION THREAD

**Discussion by DevChu** [pixijs/pixijs#10478] [Q&A] [answered]
Hi, I try to draw lines by using Graphics.
But it acts weird in 8.1.0.
Here are my examples:
v7.4.2 -> https://codepen.io/devchu/pen/vYMPZXz
v8.1.0 -> https://codepen.io/devchu/pen/zYXbzpE

I wonder if it is a bug or just my incorrect usage.
Thanks!

**Comment by bigtimebuddy** [comment, marked as answer]
The syntax for drawing Graphics changed slightly. Try this:

```js
let testRender = () => {
  const graphics = new PIXI.Graphics();
  graphics.drawRect(50, 50, 100, 100);
  graphics.fill(0xde3249);
  let offset = 0;
  for (let i = 0, endRow = 21; i < endRow; i++) {
    let yPos = 20 + offset;
    graphics.moveTo(0, yPos);
    graphics.lineTo(1000, yPos);
    offset = yPos;
  }
  graphics.stroke(0x0);
  app.stage.addChild(graphics);
};
```

See this migration guide for more information: https://pixijs.com/8.x/guides/migrations/v8?_highlight=migration#graphics-api-overhaul
